data/json/furniture_and_terrain/furniture-plumbing.json

```diff
@@ -129,7 +129,42 @@
         { "item": "hose", "count": [ 0, 2 ] },
         { "item": "cu_pipe", "count": [ 1, 4 ] },
         { "item": "scrap_copper", "count": [ 0, 2 ] },
-        { "item": "water_faucet", "count": [ 0, 1 ] }
+        { "item": "water_faucet", "count": [ 0, 1 ] },
+        { "item": "material_aluminium_ingot", "count": 2 }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_water_heater_family",
+    "name": "family–sized water heater",
+    "looks_like": "f_water_heater",
+    "description": "An insulated metal tank with a small fire used to maintain near-boiling temperatures.  There's no longer any way to power it, but any water left in the large tank from before the power went out should be pretty clean, and it could be used to store large quantities of other liquids too.",
```

Comment:
> with a small fire

Should it imply it is a gas powered heater? i don't think they are popular, also player won't be able to use one on their base, right

Reply:
They are very popular in the north–east, because they easily have 10× the heating capacity of electric (literally 45kW vs 4.5kW). Also, the existing water heater has the same description, and it isn’t tagged as an electric appliance either.